includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -2404,7 +2404,7 @@ protected function check_response_for_errors( $response ) {
 					$response_code
 				);
 			} elseif ( isset( $response_body['error'] ) ) {
-				$response_body_error_code = $response_body['error']['code'] ?? null;
+				$response_body_error_code = $response_body['error']['code'] ?? $response_body['error']['message_code'] ?? null;
```

Comment:
See the body response here https://github.com/Automattic/woocommerce-payments/issues/10025

Without checking `error['message_code']`, we can not catch this error code and pass it to `API_Exception`